modules/mauth-authenticator-apachehttp/src/main/java/com/mdsol/mauth/apache/HttpClientPublicKeyProvider.java

```diff
@@ -35,48 +37,43 @@ public class HttpClientPublicKeyProvider implements ClientPublicKeyProvider {
   private final Signer signer;
   private final CloseableHttpClient httpclient;
   private final PublicKeyResponseHandler publicKeyResponseHandler;
-
-  private LoadingCache<UUID, PublicKey> publicKeyCache;
-  private Long ttl;
+  private LoadingCache<UUID, PublicKeyData> publicKeyCache;
```

Comment:
This can be made a final now, it only needs to be set in the constructor.

Reply:
updated in https://github.com/mdsol/mauth-jvm-clients/pull/142/commits/907273382ec3d2c60482ca7168f88b6e391bd96b